pkg/cache/export_cache.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import (
+	"fmt"
+
+	"github.com/okteto/okteto/pkg/constants"
+)
+
+// ExportCache is a list of images that will be created to export the cache.
+type ExportCache []string
```

Comment:
How complicated is it to handle it as a map? I think it would improve the data handling internally (for example when adding values we don't need to check if they already exist)